client/signup/controller.js

```diff
@@ -99,10 +99,11 @@ export const removeP2SignupClassName = function () {
 export default {
 	redirectTests( context, next ) {
 		const currentFlowName = getFlowName( context.params );
-		currentFlowName === 'onboarding' && loadExperimentAssignment( 'refined_reskin_v1' );
+		[ 'onboarding', 'with-design-picker' ].includes( currentFlowName ) &&
```

Comment:
Are we enabling this experiment for the `with-design-picker` flow only for us to test it in this PR, or will this change be permanent ?

Reply:
It shouldn't be permanent since we are not planning to overlap the two experiments. See pbxNRc-LC-p2#comment-1982

But I think it's a good way to make sure the design picker is compatible with a light theme.

Reply:
Addressed in a call